src/key_finder.rs

```diff
@@ -25,8 +25,8 @@ impl KeyFinder {
 
     pub fn key<'a>(&self, record: &'a str, s: &'a mut String) -> Result<&'a str, Error> {
         let (num, last, keep) = match &self.keys {
-            None => return Ok(record.into()),
-            Some((num, _, _)) if *num == 0 => return Ok(record.into()),
+            None => return Ok(record),
```

Comment:
Let's have these unrelated changes in a separate commit or PR.

Reply:
Done.